AyOK-Code/oscn
# issue 82

**To Test**
run `rails console`
 Rails.application.load_tasks 
Rake::Task['doc:doc_facilities'].invoke

~~**Problem With Duplicates**~~
~~ran the task and noticed that there are many repeat statuses that didn't get their associated  doc_facility_id. ~~

~~using LAWTON CASELOAD (BECKHAM-CUSTER) as an example we see that in total we have 1135 statuses with that facility. 3 of those have  doc_facility_ids~~

~~on a larger scale,  there are 236 statuses that have an id.~~

~~removing the duplicate status gets us 204~~

~~those 204 where the ones that where included in the task. but it seems it was still able to pick up some extra duplicates.~~

~~in total,  there are 1128830 statuses  that have a facility~~

~~for the LAWTON CASELOAD (BECKHAM-CUSTER)  facility alone there are 1135 status with this facility, and only 3 of them have an id~~

~~should I expand my rake task to not focus on the distinct values  and open it up to the duplicates?~~




EDIT: Decided to apply rake tasks to all statuses
